x-pack/plugins/alerting/server/alert/create_alert_factory.ts

```diff
@@ -116,6 +123,22 @@ export function createAlertFactory<
           );
         }
       },
+      trimRecovered: (recoveredAlerts: TrimRecoveredOpts[]) => {
```

Comment:
nit: maybe `getEarlyRecoveredAlerts` would be a more descriptive function name?

Reply:
Resolved in this commit e33b281361df85b651c425f597d80a69193eaa52